vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -265,6 +277,17 @@ void FlyoutShadowNode::AdjustDefaultFlyoutStyle()
   m_flyout.FlyoutPresenterStyle(flyoutStyle);
 }
 
+winrt::Popup FlyoutShadowNode::GetFlyoutParentPopup() const
+{
+  // TODO: Use VisualTreeHelper::GetOpenPopupsFromXamlRoot when running against RS6
+  winrt::Windows::Foundation::Collections::IVectorView<winrt::Popup> popups = winrt::VisualTreeHelper::GetOpenPopups(winrt::Window::Current());
+  for (auto popup : popups)
+  {
+    return popup;
+  };
```

Comment:
don't need the semicolon at the end of a block.  We must be missing a compiler warning for empty controlled statement